packages/scalecube-microservice/src/Proxy/index.ts

```diff
@@ -0,0 +1,36 @@
+import { getServiceMeta } from '../helpers/utils';
+import { createDispatcher } from '../Dispatcher';
+import { Message } from '../api/Message';
+
+export const createProxy = ({ serviceRegistry, preRequest$, postResponse$, serviceContract, router }) => {
+  const meta = getServiceMeta(serviceContract);
+  const dispatcher = createDispatcher({ router, serviceRegistry, preRequest$, postResponse$ });
+
+  return new Proxy(
+    {},
+    {
+      get: validateMeta({ meta, dispatcher }),
+    }
+  );
+};
+
+const validateMeta = ({ meta, dispatcher }) => (target, prop) => {
+  if (typeof meta.methods[prop] === 'undefined') {
```

Comment:
@IdoJS maybe it would be better to use check `if(!meta.methods[prop])` Because for example `typeof null` will return "object"

Reply:
if property doesn't exist in object then it will be undefined, but we can convert to boolean checking instead :)